spacepy/pycdf/__init__.py

```diff
@@ -103,93 +105,19 @@ class Library(object):
     Calling the C library directly requires knowledge of
     `ctypes`.
 
-    Instantiating this object loads the C library, see :doc:`/pycdf` docs
+    Instantiating this object loads the C library, see :doc:`/autosummary/spacepy.pycdf` docs
```

Comment:
This should either refer to the module or to the new rst. I think probably the new rst. Never directly refer to the autosummary docs, always to objects in them.

Reply:
the new rst bring you to the additional user documentation, I am trying to get to the API documentation which is autogenerated, I think going to the module would be the best best.

Reply:
Yep, if you want the module, `~spacepy.pycdf` should do the trick.

Reply:
Gotcha, It should indeed link to the user docs, will fix this now